vscode/snippets/ei-snippets/src/main/java/org/wso2/integration/ballerina/util/CommonUtils.java

```diff
@@ -0,0 +1,570 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.wso2.integration.ballerina.util;
+
+import org.antlr.v4.runtime.CommonToken;
+import org.ballerinalang.langserver.common.CommonKeys;
+import org.ballerinalang.langserver.compiler.LSContext;
+import org.ballerinalang.langserver.completions.util.ItemResolverConstants;
+import org.ballerinalang.model.elements.PackageID;
+import org.ballerinalang.model.symbols.SymbolKind;
+import org.ballerinalang.model.types.ConstrainedType;
+import org.ballerinalang.util.BLangConstants;
+import org.eclipse.lsp4j.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BInvokableSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.types.*;
+import org.wso2.ballerinalang.compiler.tree.BLangImportPackage;
+import org.wso2.ballerinalang.compiler.tree.BLangPackage;
+import org.wso2.ballerinalang.compiler.tree.expressions.BLangExpression;
+import org.wso2.ballerinalang.compiler.util.Name;
+import org.wso2.ballerinalang.util.Flags;
+
+import java.io.File;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class CommonUtils {
+    private static final Logger logger = LoggerFactory.getLogger(CommonUtils.class);
+
+    public static final String MD_LINE_SEPARATOR = "  " + System.lineSeparator();
```

Comment:
Any specific reason to use two spaces ("  ") here?

Reply:
This redundant method was removed